bin/multielasticdump

```diff
@@ -339,15 +340,32 @@ if (options.direction === 'dump') {
     path: '/_aliases'
   }
 
+  if (options.matchType === 'datastream') {
+    req.path = '/_data_stream'
+  }
+
   elasticRequest(req, (response) => {
-    let indexes = response
-    if (!Array.isArray(response)) {
-      indexes = Object.keys(response)
+    switch (options.matchType) {
+      case 'alias': {
+        let indexes = response
+        if (!Array.isArray(response)) {
+          indexes = Object.keys(response)
+        }
+        matchedIndexes = indexes.filter(index => {
+          const aliases = Object.keys(response[index].aliases || {})
+          return matchRegExp.test(index) || aliases.some(alias => matchRegExp.test(alias))
+        })
+        break
+      }
+      case 'datastream': {
+        matchedIndexes = response.data_streams.filter(stream => matchRegExp.test(stream)).map(stream => stream.name)
```

Comment:
Change matchRegExp.test(stream)) to matchRegExp.test(stream.name))

Reply:
this is fixed.